resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -548,3 +548,33 @@ def test_umi_tools_dedup(self):
 
         dedup_paired = self.run_process('umi-tools-dedup', {'alignment': bam_paired.id})
         self.assertFile(dedup_paired, 'stats', './corall/output/dedup_paired_stats.txt')
+
+    @tag_process('seqtk-rev-complement-single', 'seqtk-rev-complement-paired')
+    def test_reverse_complement(self):
+        with self.preparation_stage():
+            reads = self.run_process('upload-fastq-single', {
+                'src': ['hs_slamseq_R1.fastq.gz']
+            })
+
+            paired_reads = self.prepare_paired_reads(['hs_slamseq_R1.fastq.gz'],
+                                                     ['hs_slamseq_R2.fastq.gz'])
+
+            inputs_single = {
```

Comment:
It makes sense to put inputs close to the process that uses them. I suggest the following syntax:

    revcomp_single = self.run_process('seqtk-rev-complement-single', {'reads': reads.id})

and:

    revcomp_paired = self.run_process('seqtk-rev-complement-paired', {
        'reads' : paired_reads.id,
        'select_mate' : 'Mate 1',
    })